install/playbooks/roles/user-setup/tasks/main.yml

```diff
@@ -141,3 +77,10 @@
     - '{{ administrators | list }}'
   loop_control:
     loop_var: user
+
+- name: Set the home folders to the group users
+  tags: accounts
+  file:
+    path: '/home/users/'
+    group: users
+    recurse: yes
```

Comment:
This task is needed to keep the group `users` because the `mkhomedir_helper` sets the group to the primary group of the user, which in ldap is set to `mail_users` (`administrators` for the postmaster account).

Is there a reason to use a single group for users? (Simplicity? Future possible feature?)

What about using a group per user (with the same uid)? Would that be a good thing, a problem, or limiting?

Is the `mail_user` ldap group used in some way?

Reply:
Ok. :)

Should the primary group be set to `users` (100) for users in ldap, and the `mail_users` group as a secondary group? Or should we use `mail_users` as user group everywhere?

Or just leave it as is, what do you think?